bigtable/bttest/inmem_test.go

```diff
@@ -410,6 +411,53 @@ func (s *MockReadRowsServer) Send(resp *btpb.ReadRowsResponse) error {
 	return nil
 }
 
+func TestCheckTimestampMaxValue(t *testing.T) {
```

Comment:
Could you add a short comment to this test to indicate what it's supposed to be verifying? You can cite the bug in question.

Reply:
Of course, done